### ISSUE TITLE
Catalogue contributions

### ISSUE THREAD

**Issue by laurenheywood** [drdvo/OWLTEH#9]
### Help us build a Catalogue of examples of the Open Web can be used in learning and teaching. 

Contributions could include applications, platforms and concepts related to working on/with the Open Web. However big or small, help others to understand how the Open Web can be used in learning teaching by sharing your expertise.

The Catalogue is a collection of instances of the Open Web covering applications and platforms that can be relevant for teaching and learning. It also includes resources and references that can **help people understand what the Open Web is**. Do you know any Open Web tools that you would like to share with colleagues around the world? Contribute via the Catalogue form here: **[http://catalogue.owlteh.org/collect/](http://catalogue.owlteh.org/collect/)**

You can find an example Catalogue contribution here: **[http://catalogue.owlteh.org/splot-tools/](http://catalogue.owlteh.org/splot-tools/)**

#### More information:

The phrase ‘Open Web‘ might be traced back to the original vision behind the creation of the World Wide Web (WWW), imagined by its creator Sir Tim Berners-Lee as [“an open platform that would allow everyone, everywhere to share information, access opportunities, and collaborate across geographic and cultural boundaries.”](https://webfoundation.org/2017/03/web-turns-28-letter/)

OWLTEH looks at online infrastructures that somehow enable this, with a particular focus on how they can be used within educational context. While the concept is often associated with open source, open standards or open licences, we welcome different interpretations of the concept and understand it as a continuum that can encompass different levels of openness.

**Comment by Daniel-Mietchen** [comment]
I have tried to think of something to submit, but had the following problems:
- the example listed above currently gives a 404
- it's not clear what is in scope here or not (see also my [related comment](https://github.com/drdvo/OWLTEH/issues/8#issuecomment-388080367) in another thread)
- You do not collect/ provide information about the licensing of the content, source code or screenshot of the applications / concepts / other resources / platforms, or even just the text of the catalog entries or the OWLTEH catalog itself (technically, your LICENCE file here in the repo only applies to the repo itself).

It would help if you could address these issues and provide some guidance as to how one would identify content suitable for your catalogue.

**Comment by mirdaki** [comment]
Hello! Stopped by to say I added information about [The Jupyter Notebook](http://catalogue.owlteh.org/the-jupyter-notebook/) and [Exercism](http://catalogue.owlteh.org/exercism/).

Another platform you might be interested in is [Matrix](https://matrix.org/blog/home/), an open network for secure, decentralized communication. It's not specifically for learning/education so I didn't add it (though there is no reason you couldn't use it for educational purposes).

**Comment by mirdaki** [comment]
I also agree with @Daniel-Mietchen. It would be beneficial to more explicitly define what "open web" means and add a clear copyright statement (like the [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/legalcode) you have on this repo) on the catalogue posts.

Edited: I saw there were licenses on the perspectives and stories (though the current stories don't show anything in the license field). Catalogue posts don't seem to have anything though.

**Comment by laurenheywood** [comment]
Hi @Daniel-Mietchen. Thanks for your feedback.

- I have now updated the original issue text to amend the broken link.
- The forms in each of the collections have a field for licensing, although this only refers to the content of the image or video contributed. @drdvo and I will look to see how we can create a guide for each of the collections to advise how best to display the license of both the referenced application / concept / resource / platform and the contributions submission itself. We will let you know how we have made changes to address this in this issue thread.

**Comment by laurenheywood** [comment]
Hi @mirdaki. Thanks for adding information about The Jupyter Notebook and Exercism, and for letting us know about [Matrix](https://matrix.org/blog/home/), I'll take a look!

Thanks for pointing out the lack of clear licensing across the sites. I'll take a look at this now.

**Comment by Daniel-Mietchen** [comment]
The more I think about your project, the more I am wondering why this information is not collected via Wikidata, where it would become available in many other contexts, and you could benefit from what is already there.

Your project could then focus on
- getting the necessary information into Wikidata
- customizing the front end for your target audience.

See https://twitter.com/EvoMRI/status/991790599698288640 for more details and https://github.com/mozilla/global-sprint/issues/237 for a Mozsprint project that handles the Wikidata part.

**Comment by Daniel-Mietchen** [comment]
Besides, the data in Wikidata is Public Domain/ CC0, which seems more appropriate for data than CC BY. Images under any open license can go to Wikimedia Commons and linked from Wikidata.

**Comment by drdvo** [comment]
@mirdaki thanks for your contributions. You can read here more on why we decided not to use a narrow definition of 'open': https://github.com/drdvo/OWLTEH/issues/8#issuecomment-388094678

The **Perspectives** and **Stories** sites are meant to gathered content produced by contributors.
In the case of **Catalogue**, it is more for signposting tools and linking to other sites. We originally had an option for contributors to choose from a set of licences, but that was referring to the image. Becasue it could get confusing (as people could think that referred to the licence of the tools added to the catalogue), we decided to disable that option and make note within the "Description" field of the form.

**Comment by drdvo** [comment]
thanks @Daniel-Mietchen. We will look in detail at Wikidata as an option. At this stage we wanted to prioritise identifying a wide number of tools and thought that setting a simple site like http://catalogue.owlteh.org/ would suffice for that purpose. But the more interesting bit would be about, later on, collecting stories (here: http://stories.owlteh.org/) of how some of those tools have been used within specific contexts. Thanks for the links! Will read in detail

**Comment by ankuhl** [comment]
@laurenheywood I added StoryEngine to the catalogue, but accidentally hit enter when entering tags and published my contribution before I was ready. 
![screen shot 2018-05-10 at 4 11 14 pm](https://user-images.githubusercontent.com/25535131/39892283-1daf2a84-546e-11e8-82a2-467f6dd6411d.png)
When I click on the "Edit post" I am redirected to http://connectedcourses.stateu.org/seeds/collector/

**Comment by laurenheywood** [comment]
Thanks @ankuhl for adding StoryEngine to the Catalogue. Let me know what tags you would like to add and I can make these changes to your contribution. I'll take a look at the misdirecting link tomorrow morning and get back to you. If you have other changes or questions let me know.

**Comment by ankuhl** [comment]
@laurenheywood I also need to edit the description, as I hadn't finished proof-reading and there are mistakes. Another option: Can you take down the listing and I can resubmit it?

**Comment by laurenheywood** [comment]
@ankuhl yes I have taken the original submission down and you can now resubmit.

**Comment by drdvo** [comment]
thanks @ankuhl for reporting the Glitch. I will check with @cogdog to see how we can fix it.

**Comment by laurenheywood** [comment]
I've made changes to the original issue text inline with comments we have received asking for a clearer definition of what is being asked for. We have had non-GitHub folk asking for this as they were not sure what contributions would be appropriate. If anyone has further queries or has suggestions on how we can improved this request description, please let us know 🙌
